### DISCUSSION TITLE
некорректное отображение звонков в журналах

### DISCUSSION THREAD

**Discussion by YanchikAmb** [mikopbx/Core#66] [Q&A] [answered]
имеем настроенное правило исходящего звонка
![изображение](https://user-images.githubusercontent.com/9751378/107637415-27d74d80-6c7f-11eb-87ec-d5d13aee0e4e.png)
звонок проходит и до конечного абонента мы дозваниваемся, но в журналах звонков отображается только то, что мы просили добавить в правиле, без остальных цифр номера.
![изображение](https://user-images.githubusercontent.com/9751378/107637562-5b19dc80-6c7f-11eb-8271-dffc1e9d17f7.png)

[verbose.zip](https://github.com/mikopbx/Core/files/5965219/verbose.zip)

**Comment by boffart** [comment, marked as answer]
Это баг. Уже исправлен в develop ветке
Проблема в описании extension _.! включает в себя и h служебный exten.
Нужно **_.!** заменить на **_[0-9*#+]**
